src/index.css

```diff
@@ -1,9 +1,7 @@
 :root {
   font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
-
   background-color: #242424;
-
```

Comment:
Personally I would prefer to keep some blank lines to separate different groups of properties.